src/server/src/routers/file-io/providers/write-file.ts

```diff
@@ -0,0 +1,26 @@
+import { Request, Response } from 'express';
+import * as fs from 'fs';
+import * as path from 'path';
+
+import { getFileNameByExtension } from '../../../utils/getFileName';
```

Comment:
Need to use webpack for bundling, or we can create a node-modules folder in order to trick node to find our aliased folder.